x-pack/plugin/rollup/src/main/java/org/elasticsearch/xpack/rollup/action/TransportGetRollupCapsAction.java

```diff
@@ -26,21 +28,37 @@
 import java.util.Map;
 import java.util.Optional;
 import java.util.TreeMap;
+import java.util.concurrent.Executor;
 import java.util.stream.Collectors;
 
 public class TransportGetRollupCapsAction extends HandledTransportAction<GetRollupCapsAction.Request, GetRollupCapsAction.Response> {
 
     private final ClusterService clusterService;
+    private final ThreadPool threadPool;
+    private final Executor managementExecutor;
 
     @Inject
-    public TransportGetRollupCapsAction(TransportService transportService, ClusterService clusterService, ActionFilters actionFilters) {
+    public TransportGetRollupCapsAction(
+        TransportService transportService,
+        ClusterService clusterService,
+        ThreadPool threadPool,
+        ActionFilters actionFilters
+    ) {
         super(GetRollupCapsAction.NAME, transportService, actionFilters, GetRollupCapsAction.Request::new, ThreadPool.Names.MANAGEMENT);
```

Comment:
@DaveCTurner should I use `ThreadPool.Names.SAME` here, like we do in 

https://github.com/elastic/elasticsearch/blob/2b8ccf4ca02805555b0927aa35bc4e708427bcc9/server/src/main/java/org/elasticsearch/action/support/broadcast/node/TransportBroadcastByNodeAction.java#L110

Reply:
In practice it doesn't matter in this case because this action is only ever executed from REST actions, which ignore this parameter, but yes it'd be good to follow the same pattern anyway. Please also include the comment about https://github.com/elastic/elasticsearch/issues/97916.